activities.json

```
@@ -252,6 +252,19 @@
     "title": "Contact Picker API",
     "url": "https://wicg.github.io/contact-api/spec/"
   },
+  {
+    "ciuName": null,
+    "description": "This specification extends MediaStreamTrack to let web applications provide an optional media-content hint attribute. This helps sinks like RTCPeerConnection or MediaRecorder select encoder parameters and processing algorithms appropriately based on a hint about the nature of the content being consumed without having to examine the actual content.",
+    "id": "content-hints",
```

```suggestion
    "id": "mst-content-hint",
```

For W3C drafts, I tend to prefer using the TR shortname as the `id` unless there's a good reason not to.  ("content-hints" also sounds a bit more general.)